release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -139,6 +139,17 @@ The Network Observability Operator releases updates independently from the {prod
 [id="ocp-4-16-scalability-and-performance"]
 === Scalability and performance
 
+[id="workload-partitioning-enhancement"]
+=== Workload partitioning enhancement
+With this release, if you have a pod deployed with a workload annotation and it contains both CPU limits and CPU requests, the CPU limits in the annotation get correctly calculated and applied as a CPU quota for that specific pod. In prior releases, the CPU limits and requests were ignored by the webhook, the pod did not benefit from workload partitioning and was not locked down to specific cores. This update ensures the requests and limits are now interpreted correctly by the webhook. 
```

Comment:
Small nit, since customers don't really have access to this feature yet, maybe changing this to something like. 

Change
`With this release, if you have a pod deployed with` > `With this release, platform pods deployed with`

`In prior releases, the CPU limits and requests were ignored by the webhook,` > `In prior releases, if a workload partitioned pod had both CPU limits and requests set, they were ignored by the webhook,`

Reply:
@eggfoobar Let me know if the changes are OK and add a LGTM if happy with the update.